prac_08/box_layout_demo.py

```diff
@@ -0,0 +1,21 @@
+from kivy.app import App
+from kivy.lang import Builder
+
+
+class BoxLayoutDemo(App):
+    def build(self):
+        self.title = "Box Layout Demo"
+        self.root = Builder.load_file('box_layout.kv')
+        return self.root
+
+    def handle_greet(self):
+        print("test")
+        self.root.ids.output_label.text = f"Hello {self.root.ids.input_name.text}"
+
+    def handle_clear(self):
+        print("test")
+        self.root.ids.output_label.text = "Enter your name"
```

Comment:
You may use an empty string instead. The clear method should reset both the text field and the output label to blank.